docs/kubernetes/ingress-controller.md

```diff
@@ -281,3 +281,119 @@ Example ingress:
           - backend:
               serviceName: app-svc
               servicePort: 80
+
+## Install Skipper with enabled RBAC
+
+If Role-Based Access Control ("RBAC") is enabled you have to create some additional
+resources to enable Skipper to query the Kubernetes API.
+
+This guide describes all necessary resources to get Skipper up and running in a
+Kubernetes cluster with RBAC enabled but it's highly recommended to read the
+[RBAC docs](https://kubernetes.io/docs/admin/authorization/rbac/) to get a better
+understanding which permissions are delegated to Skipper within your Kubernetes cluster.
+
+First create a new `ServiceAccount` which will be assigned to the Skipper pods:
+
+    apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name:  aws-ingress-controller
+      namespace: kube-system
+
+the required permissions are defined within a `ClusterRole` resource.
+
+_Note: It's important to use a `ClusterRole` instead of normal `Role` because otherwise Skipper could only access resources in the namespace the `Role` was created!_
+
+ClusterRole:
+
+    apiVersion: rbac.authorization.k8s.io/v1beta1
+    kind: ClusterRole
+    metadata:
+      name: aws-ingress-controller
+    rules:
+    - apiGroups: ["extensions"]
+      resources: ["ingresses", ]
+      verbs: ["get", "list"]
+    - apiGroups: ["extensions"]
+      resources: ["ingresses/status", ]
+      verbs: ["get", "list", "patch"]
```

Comment:
Are you sure we need "patch", I think we don't need "patch". Skipper only reads, and let for example the kube-aws-ingress-controller do the update of the status field.

Reply:
I think we also do not need ingresses/status api group.

Reply:
Removed `ingresses/status` I think it's only required for kube-ingress-aws-controller